assets/_locales/de/messages.json

```diff
@@ -939,6 +953,10 @@
     "message": "Zeige den Button um einen Anime/Manga auf MAL oder einem anderen Anbieter anzusehen",
     "description": "Show button to view the anime/manga on MAL or another provider"
   },
+  "settings_presenceShowMalsync": {
+    "message": "Zeige Mal-Sync im Titel der Presence anstatt des Anime/Manga",
```

Comment:
It's literally shown as "Anime" or "Manga" in the presence so I think this sentence would confuse the user that it shows the anime title

Reply:
Idk but isn't that a near 1:1 translation? "Show Mal-Sync in the title of the presence instead of anime/manga"